### PR TITLE
Blog - Analyze Your Amazon Data with Python

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by jalletto** [comment]
Commit 1f9088a40b132c55d0f1104d4c4854b2dc62e9cc deployed to https://62b9d279f345055eb1b450f8--ephemeral-narwhal-3ac3b2.netlify.app/

**Review by jalletto — APPROVED** [review]

File: blog/_posts/2022-07-19-analyze-amzndata-pyth.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+* A Python IDE and Python 3.x
+* [Pandas](/blog/plotting-rainfall-data-with-python-and-matplotlib) Library (It is a built-in library but in case it is not right there, install it through `pip install pandas`)
+* Matpotlib installed (to install, use `pip install matpotlib`)
+* Amazon Order History data; Follow these steps to download it!
+
+Step 1: Go to Amazon.com and sign in to your account.
+
+Step 2: Click on Your Account > Account
+
+![Create account]({{site.images}}{{page.slug}}/bjGElgm.png)\
```
Comment (adamgordonbell) [inline blog/_posts/2022-07-19-analyze-amzndata-pyth.md:38]:
```
<div class="wide">
![Create account]({{site.images}}{{page.slug}}/bjGElgm.png)\
</div> 
```

File: blog/_posts/2022-07-19-analyze-amzndata-pyth.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+Step 3: In the Order and Shopping Preferences section, select "Download order reports"
+
+![download order report]({{site.images}}{{page.slug}}/NQdSCrv.png)\
+Step 4: Customize the data according to your need and click on "Request Report".
```
Comment (adamgordonbell) [inline blog/_posts/2022-07-19-analyze-amzndata-pyth.md:42]: `<div class="wide">`

File: blog/_posts/2022-07-19-analyze-amzndata-pyth.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+
+![request report]({{site.images}}{{page.slug}}/KJilq9p.png)\
+Congrats! You've your order history stored in a CSV file. Now we can move to code.
```
Comment (adamgordonbell) [inline blog/_posts/2022-07-19-analyze-amzndata-pyth.md:45]: `<div class="wide">`

File: blog/_posts/2022-07-19-analyze-amzndata-pyth.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+
+df["Item Total"] = df["Item Total"].str.replace('$','').astype(float)
+bar = df.plot.bar(x='Order Date', y='Item Total', rot=0, color="#36b6fa")
+
+plt.show()
+~~~
+
+Run the code, and there will be an awesome bar chart like the below:
+
+![Bar Chart]({{site.images}}{{page.slug}}/zrFvTZw.png)\
```
Comment (adamgordonbell) [inline blog/_posts/2022-07-19-analyze-amzndata-pyth.md:377]: `<div class="wide">`

File: blog/_posts/2022-07-19-analyze-amzndata-pyth.md (hunk 2 above)
Comment (msali123) [reply blog/_posts/2022-07-19-analyze-amzndata-pyth.md:42]: @adamgordonbell  I've done the changes they'll come up with the new article

Comment (adamgordonbell) [reply blog/_posts/2022-07-19-analyze-amzndata-pyth.md:42]: Thanks @msali123 !

**Pull request merged**